projects/media-viewer/src/lib/bookmark/components/bookmark.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Bookmark} from '../../viewers/pdf-viewer/side-bar/bookmarks/bookmarks.interfaces';
+import {Component, Input, OnDestroy, OnInit} from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import * as fromStore from '../../store/reducers/reducers';
+import * as fromSelectors from '../../store/selectors/bookmarks.selectors';
+import {Observable, Subscription} from 'rxjs';
+
+@Component({
+  selector: 'mv-bookmark',
+  templateUrl: './bookmark.component.html'
+})
+export class BookmarkComponent implements OnInit, OnDestroy {
+
+  @Input() zoom: number;
+  @Input() rotate: number;
+  bookmarksPerPage$: Observable<any>; // todo add type
```

Comment:
you could add the type

Reply:
Could you point me at any examples for this please?

Reply:
all you need to do is create an interface based on what that observable streams  and put it instead of <any> ping me directly if you need help to find out what it streams